src/nvim/msgpack_rpc/channel.c

```diff
@@ -731,101 +535,44 @@ static void unsubscribe(Channel *channel, char *event)
   xfree(event_string);
 }
 
-/// Close the channel streams/process and free the channel resources.
-static void close_channel(Channel *channel)
+
+/// Release the rpc infastructures reference to the channel
```

Comment:
typo: infastructures -> infrastructure
`rpc` is released(freed?) with the channel, perhaps change comment to something like  `Mark rpc as closed, prepare to free channel.`